requirements/bench.txt

```diff
@@ -14,3 +14,4 @@ torchvision
 pyomo>=5.7
 IOHexperimenter>=0.2.8
 mixsimulator>=0.2.9.5
+git+git://github.com/pacowong/pypownet@master#egg=pypownetr
```

Comment:
that ends up being a pain to install with the CI, this should not be added here. The best option would be to import inline and provide an explicit error message if not importabe.